pyqir-generator/src/python.rs

```diff
@@ -30,139 +30,125 @@ impl PyQIR {
         }
     }
 
-    fn add_measurement(&mut self, qubit: String, target: String) -> PyResult<()> {
+    #[allow(clippy::needless_pass_by_value)]
+    #[allow(clippy::unused_self)]
+    fn add_measurement(&mut self, qubit: String, target: String) {
```

Comment:
Will it be a breaking change to the API if you need to return an Err in the future?